faunadb-java-dsl/src/test/java/com/faunadb/client/database/SetupDatabase.java

```diff
@@ -0,0 +1,30 @@
+package com.faunadb.client.database;
+
+import com.faunadb.client.query.Expr;
+import com.faunadb.client.types.Value;
+
+import static com.faunadb.client.query.Language.*;
+import static com.faunadb.client.types.Codec.REF;
+
+public class SetupDatabase {
```

Comment:
I know I sort of asked for this class before. But looking at it now I feel like there isn't much value on extracting so little code out of the drivers test.

Feel free to inline those methods if you feel like. I'll let you decide.